### PR TITLE
Feature/character movement

### PR DESCRIPTION
**Pull request by LickMyPoppy**
Implemented character movement.
The character can walk, run, jump, crouch and dodge(roll)

### CODE REVIEW

File: Assets/Scripts/CharacterMovement.cs [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    }
+    [SerializeField]
+    public MovementSettings movement;
+
+    //private variables
+    private bool jumping;
+    private bool dodging;
+    private bool crouching;
+    private float speed;
+    //private float originalColliderSize;
```
Comment (bitingbits) [inline Assets/Scripts/CharacterMovement.cs]: remove commented out code

File: Assets/Scripts/CharacterMovement.cs [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    //Bool to check if the player is rolling.
+    
+
```
Comment (bitingbits) [inline Assets/Scripts/CharacterMovement.cs]: fix indentation [CLEAN YO FILE]

File: Assets/Scripts/CharacterMovement.cs [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    {
+        float distToGround = 0.1f;
+        return Physics.Raycast(transform.position, -Vector3.up, distToGround);
+    }
+
+    void Start()
+    {
+        animator = this.transform.GetComponent<Animator>();
+        characterController = GetComponent<CharacterController>();
+        //originalColliderSize = characterController.height;
```
Comment (bitingbits) [inline Assets/Scripts/CharacterMovement.cs]: commented out code, remove

File: Assets/Scripts/CharacterMovement.cs [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (IsGrounded())
+        {
+            if (Input.GetButton("Crouch"))
+            {
+                crouching = true;
+                //characterController.height = originalColliderSize / 2;
```
Comment (bitingbits) [inline Assets/Scripts/CharacterMovement.cs]: commented out code, remove

File: Assets/Scripts/CharacterMovement.cs [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+{
+
+    Animator animator;
+    CharacterController characterController;
+    
+    //Serialized classes
+    [System.Serializable]
+    public class AnimationSettings
+    {
+        public string verticalVelocityFloat = "Forward";
```
Comment (Caprico1) [inline Assets/Scripts/CharacterMovement.cs]: Place a comment here to show that these are the 'fields' where values are stored in the animator. Naming a variable as float will be confusing as you've typed it a string.

File: Assets/Scripts/CharacterMovement.cs [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+                StartCoroutine(Roll(false, Input.GetAxis("Vertical")));
+            }
+        }
+    }
+
+    IEnumerator Roll(bool horizontal, float direction)
+    {
+        print("dodging");
+        dodging = true;
+        /* not needed anymore
```
Comment (Caprico1) [inline Assets/Scripts/CharacterMovement.cs]: Remove this if not needed.

**Pull request merged**